### DISCUSSION TITLE
v2.1.4

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3027] [分享 / Share]
- 修复对 UP 主视频合集页的支持. (#2986, PR #2995 by [timongh](https://github.com/timongh))
- 修复使用 `播放全部` 进入稍后再看时功能不生效. (#2339, #2986, PR #2995 by [timongh](https://github.com/timongh))
- 修复 `图片批量导出` 没过滤掉表情图片. (#2943)
- 更换了 cid Hook 的方式, 能够更好地兼容 Bilibili-Old 等脚本. (#3005)
- 修复 XML 弹幕导出的内容没转义. (#3003)
- `稍后再看重定向` 在生成链接时, 自动省略 `p=1` 的参数. (#2989)
- 修复部分番剧下载时命名错误. (#2988)
- 修复 `网址参数清理` 导致直播间热门榜出现问题. (#2953)
- `传统连播模式` 支持判断合集. (#2940)
- 修复 `简化直播间` 的房间皮肤屏蔽功能失效. (#2726)
- 修复 `自定义顶栏` 中的直播间标题过长时布局错乱. (#2886)
- `夜间模式` 支持视频页中的合集选择区. (#2136)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.4'>v2.1.4</a>.</em>

**Comment by Geeyun-JY3** [comment]
视频播放页的合集选择区情况好像还挺多的：
- https://www.bilibili.com/video/av57114947
![image](https://user-images.githubusercontent.com/49396421/154828449-f0b6f493-911f-458f-ba0f-caf370c9e5d8.png)
- https://www.bilibili.com/video/av52357487
![image](https://user-images.githubusercontent.com/49396421/154828395-8f542b2f-8225-4ad3-8108-b7a666fef835.png)

**Reply by xzk-kokodayo** [reply]
夜间模式还有问题
